meteor/client/ui/SegmentTimeline/RundownTimeline.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import * as _ from 'underscore'
+import { Rundown } from '../../../lib/collections/Rundowns'
+
+interface IProps {
+	key: string
+	rundown: Rundown
+}
+interface IState {}
+export class RundownDividerTimeline extends React.Component<IProps, IState> {
```

Comment:
See my coment above regarding the naming of this component. Also, I don't think it needs the state, so maybe this could be a Stateless Function Component?